pom.xml

```diff
@@ -571,7 +571,7 @@
       <dependency>
         <groupId>io.netty</groupId>
         <artifactId>netty-all</artifactId>
-        <version>4.0.43.Final</version>
+        <version>4.0.47.Final</version>
```

Comment:
Upgrade Netty version 4.0.47 which has the public API for `ByteBufAllocatorMetric`, what do you think @zsxwing ?